nfs/src/rpc_task.cpp

```diff
@@ -2660,6 +2664,12 @@ static void readdir_callback(
     int num_dirents = 0;
     const int status = task->status(rpc_status, NFS_STATUS(res));
 
+    // Last valid offset seen by this callback.
+    off_t last_valid_offset = task->rpc_api->readdir_task.get_offset();
+    
+    // We only want to send to fuse when we see a new entry.
+    bool has_new_entry = false;
```

Comment:
rename this to should_send_response so that we can then use the same variable for eof case as well